shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/util/PipelineSchemaTableUtil.java

```diff
@@ -71,19 +72,18 @@ public static Map<String, List<String>> getSchemaTablesMapFromActual(final Pipel
     }
     
     /**
-     * get default schema by connection.getSchema().
+     * Get default schema by connection.getSchema().
      *
-     * @param pipelineDataSourceConfig pipeline data source config
+     * @param dataSourceConfig pipeline data source config
      * @return schema
      */
-    public static String getDefaultSchema(final PipelineDataSourceConfiguration pipelineDataSourceConfig) {
-        try (PipelineDataSourceWrapper dataSource = PipelineDataSourceFactory.newInstance(pipelineDataSourceConfig)) {
+    @SneakyThrows(SQLException.class)
```

Comment:
`@SneakyThrows(SQLException.class)` could be improved later, maybe we could throw SQLException on upper layer `createJobAndStart`